build.gradle

```diff
@@ -167,6 +168,9 @@ subprojects {
     "org.elasticsearch.plugin:parent-join-client:${version}": ':modules:parent-join',
     "org.elasticsearch.plugin:percolator-client:${version}": ':modules:percolator',
   ]
+  if (wireCompatVersions[-1].snapshot) {
```

Comment:
can you leave a comment what this is doing? it helps me to understand this better